include/swift/SIL/SILInstruction.h

```diff
@@ -816,11 +816,11 @@ inline SingleValueInstruction *SILNode::castToSingleValueInstruction() {
            inst->getKind() <= SILInstructionKind::Last_##ID;    \
   }
 
-/// A single value inst that also forwards either owned or guaranteed ownership.
+/// A single value inst that forwards a static ownership from one (or all) of
+/// its operands.
```

Comment:
How can multiple values be forwarded to a single value? All but one of the incoming value's ownership will be lost.

Reply:
That is incorrect. If I have all owned arguments, then nothing is lost. If I have multiple owned arguments with 1 trivial value, then I only lost one incoming value's ownership kind.

Reply:
*That being said, do you have a better way of phrasing this that would be clearer to you?

Reply:
Maybe the way to say this is it forwards after merging a subset of the arguments? That is really what I was trying to say.